content/docs/deployment/offline/modules/_index.md

```diff
@@ -110,11 +110,11 @@ To perform an offline installation of a Helm chart, the following steps should b
     *
     * Loading, tagging, and pushing Docker images to registry <my-registry>:5000/
 
-      dellemc/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.6.0
-      dellemc/csm-metrics-powerflex:v1.6.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.6.0
-      dellemc/csm-metrics-powerstore:v1.6.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.6.0
-      dellemc/csm-metrics-powerscale:v1.3.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.3.0
-      dellemc/csm-metrics-powermax:v1.1.0 -> <my-registry>:5000/csm-metrics-powermax:v1.1.0
+      quay.io/dell/container-storage-modules/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.10.0
```

Comment:
First image needs to refer to v1.10.0 ->

Reply:
Yes, lets go with latest version alone.